spec/SUMMARY.md

```diff
@@ -1,15 +1,17 @@
 # Table of contents
 
-* [\<name of building block>](README.md)
+* [Information Mediator](README.md)
 * [1 Version History](1-version-history.md)
 * [2 Description](2-description.md)
 * [3 Terminology](3-terminology.md)
 * [4 Key Digital Functionalities](4-key-digital-functionalities.md)
 * [5 Cross-Cutting Requirements](5-cross-cutting-requirements.md)
 * [6 Functional Requirements](6-functional-requirements.md)
-* [7 Data Structures](7-data-structures.md)
-* [8 Service APIs](8-service-apis.md)
-* [9 Workflows](9-workflows.md)
-* [10 Other Resources](10-other-resources.md)
-* [11 Key Decision Log](11-key-decision-log.md)
-* [12 Future Considerations](12-future-considerations.md)
+* [7 Requirements Clarification ](7-requirements-clarification.md)
+* [8 Performance Requirements](8-performance-requirements.md)
+* [9 Data Structures](9-data-structures.md)
+* [10 Service APIs](10-service-apis.md)
+* [11 Workflows](11-workflows.md)
+* [13 Future Considerations](13-future-considerations.md)
```

Comment:
I don't really understand why there isn't a 12 here - there wasn't in the original?

Reply:
@rachellawson , i think that section 12 is "Key Decision Log". Is that standard? I see it in the current IM spec.

Reply:
hmm - it seems it was missing from the migration from doc to original markdown. Now added in at this stage!